drivers/lte_link_control/Kconfig

```diff
@@ -11,6 +11,10 @@ menuconfig LTE_LINK_CONTROL
 
 if LTE_LINK_CONTROL
 
+config LTE_LINK_CONTROL_LOG_LEVEL
```

Comment:
You could use logger template here, just as other modules do:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/5f577bbb2f40d0f0f83c3a860692aa029068c812/subsys/net/lib/nrf_cloud/Kconfig#L48